src/keys.rs

```diff
@@ -492,6 +580,20 @@ impl DiversifiedTransmissionKey {
     }
 }
 
+impl Default for DiversifiedTransmissionKey {
+    fn default() -> Self {
+        DiversifiedTransmissionKey(NonIdentityPallasPoint::default())
+    }
+}
+
+impl ConditionallySelectable for DiversifiedTransmissionKey {
+    fn conditional_select(a: &Self, b: &Self, choice: Choice) -> Self {
+        DiversifiedTransmissionKey(NonIdentityPallasPoint::conditional_select(
+            &a.0, &b.0, choice,
+        ))
+    }
+}
+
```

Comment:
We could remove these impls if we get rid of the `map` from my above suggestion.

```suggestion

```